woocommerce-services.php

```diff
@@ -651,12 +652,17 @@ public function woocommerce_payment_gateways( $payment_gateways ) {
 		 * Registers the React UI bundle
 		 */
 		public function admin_enqueue_scripts() {
-			wp_register_style( 'noticons', plugins_url( 'assets/stylesheets/noticons.css', __FILE__ ), array(), '20150727' );
-			wp_register_style( 'dashicons', plugins_url( 'assets/stylesheets/dashicons.css', __FILE__ ), array(), '20150727' );
+			// Note: This will break outside of wp-admin, if/when we put user-facing JS/CSS we'll have to figure out another way to version them
```

Comment:
We could just hardcode a date-based version string for each registered script and style. For example, a version of `20170218` or `1487454198`.

This has worked well for me in the past, and allows us to update versions only for scripts and styles that change between releases. It also doesn't require plugin file header parsing on every admin request.

What do you think?